Sources/Tests/CoreTests/test_hsDarwin_NS.mm

```diff
@@ -47,55 +47,45 @@
 
 TEST(hsDarwin_Foundation, converts_to_ST_string)
 {
-    NSAutoreleasePool* pool = [[NSAutoreleasePool alloc] init];
+    hsAutoreleasingScope;
```

Comment:
One option with this scope type is we could just turn off ARC in places where the code needs to be shared. I'm not sure what your plan is with the client in since that targets a newer SDK.